### PR TITLE
update create swarm and add nodes to use the auto-generated join command

### PR DESCRIPTION
**Pull request by sfsmithcha**
Tutorial should use the command output by `docker swarm init` because it is the easiest way to join nodes.

### CODE REVIEW

File: docs/swarm/swarm-tutorial/add-nodes.md [hunk 1]
```diff
@@ -19,49 +19,51 @@ to add worker nodes.
 1. Open a terminal and ssh into the machine where you want to run a worker node.
 This tutorial uses the name `worker1`.
 
-2. Run the following command to create a worker node joined to
-the existing swarm:
+2. Run the command produced by the `docker swarm init` output from the previous
+step to create a worker node joined to the existing swarm:
```
Comment (aaronlehmann) [inline docs/swarm/swarm-tutorial/add-nodes.md]: Nit: the previous step didn't involve running `docker swarm init`. Maybe "...produced by the `docker swarm init` command that created the swarm"?

File: docs/swarm/swarm-tutorial/add-nodes.md [hunk 2]
```diff
@@ -19,49 +19,51 @@ to add worker nodes.
...
+docker swarm join --secret <SECRET> <MANAGER-IP>:<PORT>
+```
 
-    ```
-    $ docker swarm join --secret 4ao565v9jsuogtq5t8s379ulb 192.168.99.100:2377
-
-    This node joined a Swarm as a worker.
-    ```
+Replace `<SECRET>` with the secret that was printed by `docker swarm init`
+in the previous step. Replace `<MANAGER-IP>` with the address of the manager
```
Comment (aaronlehmann) [inline docs/swarm/swarm-tutorial/add-nodes.md]: Same comment about "previous step".

File: docs/swarm/swarm-tutorial/add-nodes.md [hunk 3]
```diff
@@ -19,49 +19,51 @@ to add worker nodes.
...
+node and `<PORT>` with the port where the manager listens.
 
 3. Open a terminal and ssh into the machine where you want to run a second
 worker node. This tutorial uses the name `worker2`.
 
-4. Run `docker swarm join --secret <SECRET> <MANAGER-IP>:<PORT>` to create a worker node joined to
-the existing Swarm.
+4. Run the command produced by the `docker swarm init` output from the prvious
```
Comment (aaronlehmann) [inline docs/swarm/swarm-tutorial/add-nodes.md]: previous is misspelled - and the same point about the previous step applies

**Comment by SvenDowideit** [comment]
I'm kinda missing a reminder where the `--ca-cert` value came from, but otherwise :)

**Comment by SvenDowideit** [comment]
@sfsmithcha needs rebase :)

**Comment by sfsmithcha** [comment]
@aaronlehmann , @SvenDowideit thanks for the great feedback PTAL

File: docs/swarm/swarm-tutorial/create-swarm.md [hunk 4]
```diff
@@ -22,63 +22,71 @@ node. For example, the tutorial uses a machine named `manager1`.
...
+    Swarm initialized: current node (dxn1zf6l61qsb1josjja83ngz) is now a
+    manager.
 
     To add a worker to this swarm, run the following command:
-        docker swarm join --secret 4ao565v9jsuogtq5t8s379ulb \
-        --ca-hash sha256:07ce22bd1a7619f2adc0d63bd110479a170e7c4e69df05b67a1aa2705c88ef09 \
-        192.168.99.100:2377
+	    docker swarm join --secret 4ao565v9jsuogtq5t8s379ulb \
+	    --ca-hash sha256:07ce22bd1a7619f2adc0d63bd110479a170e7c4e69df05b67a1aa2705c88ef09 \
+	    192.168.99.100:2377
```
Comment (thaJeztah) [inline docs/swarm/swarm-tutorial/create-swarm.md]: looks like there's tabs here 😇

File: docs/swarm/swarm-tutorial/create-swarm.md [hunk 5]
```diff
@@ -22,63 +22,71 @@ node. For example, the tutorial uses a machine named `manager1`.
...
-    In the tutorial, the following command creates a swarm on the `manager1` machine:
+    In the tutorial, the following command creates a swarm on the `manager1`
+    machine:
 
-    ```
+    ```bash
     $ docker swarm init --listen-addr 192.168.99.100:2377
     No --secret provided. Generated random secret:
-        4ao565v9jsuogtq5t8s379ulb
+	    4ao565v9jsuogtq5t8s379ulb
```
Comment (thaJeztah) [inline docs/swarm/swarm-tutorial/create-swarm.md]: and here

File: docs/swarm/swarm-tutorial/create-swarm.md (hunk 4 above)
Comment (sfsmithcha) [inline docs/swarm/swarm-tutorial/create-swarm.md]: b/c it was cp/paste from the tabbed cli output :(

File: docs/swarm/swarm-tutorial/create-swarm.md [hunk 6]
```diff
@@ -22,63 +22,71 @@ node. For example, the tutorial uses a machine named `manager1`.
...
+    Swarm initialized: current node (dxn1zf6l61qsb1josjja83ngz) is now a
+    manager.
 
     To add a worker to this swarm, run the following command:
-        docker swarm join --secret 4ao565v9jsuogtq5t8s379ulb \
-        --ca-hash sha256:07ce22bd1a7619f2adc0d63bd110479a170e7c4e69df05b67a1aa2705c88ef09 \
-        192.168.99.100:2377
+      docker swarm join --secret 4ao565v9jsuogtq5t8s379ulb \
+      --ca-hash sha256:07ce22bd1a7619f2adc0d63bd110479a170e7c4e69df05b67a1aa2705c88ef09 \
+	    192.168.99.100:2377
```
Comment (aaronlehmann) [inline docs/swarm/swarm-tutorial/create-swarm.md]: There's still a tab on this line.

Comment (sfsmithcha) [inline docs/swarm/swarm-tutorial/create-swarm.md]: ok. updated my editor to show tabs. sorry for that.

**Comment by sfsmithcha** [comment]
@aaronlehmann , @thaJeztah , thanks for all your help with this. can you let me know what remains to get this merged? thanks.

**Comment by aaronlehmann** [comment]
LGTM

**Comment by thaJeztah** [comment]
LGTM, thanks again, @sfsmithcha !

**Pull request merged**